src/test/kotlin/org/rust/ide/annotator/fixes/AddAttrParenthesesFixTest.kt

```diff
@@ -5,10 +5,10 @@
 
 package org.rust.ide.annotator.fixes
 
-import org.rust.ide.annotator.RsAnnotatorTestBase
-import org.rust.ide.annotator.RsErrorAnnotator
+import org.rust.ide.inspections.RsAttrWithoutParenthesesInspection
+import org.rust.ide.inspections.RsInspectionsTestBase
 
-class AddAttrParenthesesFixTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
+class AddAttrParenthesesFixTest : RsInspectionsTestBase(RsAttrWithoutParenthesesInspection::class) {
```

Comment:
Let's move all tests from this class to `RsAttrWithoutParenthesesInspectionTest`